src/components/20-molecules/dropdown/index.js

```diff
@@ -15,6 +15,9 @@ const ARROW_ICON = svg([ExpandSvg]);
 const DEBOUNCE_DELAY = 250; // milliseconds
 const DROPDOWN_UL_MAXHEIGHT = '200px';
 const EMPTY_FUNCTION = () => {};
+const WORD_END = '\x00';
+const INDEX_END = '\x01';
+const AUTOSUGGEST_INACTIVITY_DELAY = 200; // milliseconds
```

Comment:
Is this the default limit? I fell like this is a bit short. When testing it out, if often did not work, maybe because i was not fast enough.

Reply:
Yes, i wasnt fast enough many times, too. This feature is a bit tricky. What do you think about a timer on every keystroke? So every stroke has a new timeout and the user is able to type long words without a "global timeout". Or isnt that the default behaviour for dropdowns?

Reply:
This has been modelled after the timing behaviour of native &lt;select&gt; which indeed requires fast typing.

Reply:
I increased to 300ms. Experiments with small screen widths - where native select is active - on Storybook show that UX is 'tricky' there as well.

Reply:
And I don't understand the timer-on-every-keystroke comment: isn't that exactly what happens in handleAutosuggest?